v2/slogger/logger.go

```diff
@@ -36,6 +36,10 @@ type Log struct {
 	MessageFmt string
 	Args       []interface{}
 	Context    *Context
+
+	// Appendages to evaluate a message only once
+	message string
```

Comment:
What is the lifecycle of a `Log` struct? We've now ~doubled the effective size of it, assuming `MessageFmt` (and now `message`) is long compared to the other fields...